lib/app/todo/item.ex

```diff
@@ -3,7 +3,7 @@ defmodule App.Todo.Item do
   import Ecto.Changeset
 
   schema "items" do
-    field :person_id, :integer, default: 0
+    field :person_email, :string, default: ""
```

Comment:
I don't understand _why_ this is necessary ... 🤷‍♂️

Reply:
Fixed with https://github.com/dwyl/phoenix-todo-list-tutorial/pull/56/commits/c9cb6313fe3fbc0e1afde401b3473155fced9613